examples/game/breakout.rs

```diff
@@ -63,7 +72,7 @@ fn setup(
             ..Default::default()
         })
         .with(Ball {
-            velocity: 400.0 * Vec3::new(0.5, -0.5, 0.0).normalize(),
+            velocity: 6.0 * Vec3::new(0.5, -0.5, 0.0).normalize(),
```

Comment:
This is still "not velocity" with the new implementation. You effectively pre-multiplied TIME_STEP and velocity. This has the effect of changing object speed if someone changes the timestep value here: https://github.com/bevyengine/bevy/pull/1541/files#diff-4dcbfd17ebad2621b5605be8bd3b43859376e27f30793a937858d470627ed2b2R18

Reply:
movement speed should be timestep independent

Reply:
Whoops.  I shouldn't read comments right before I fall asleep.  Should be fixed now.